### ISSUE TITLE
Custom CSS overwrite?

### ISSUE THREAD

**Issue by FluffyDiscord** [hazuki0x0/YuzuBrowser#185]
Hi, how hard it would be to add option to inject css per domain/page as we already can with Javascript? Also option to put it either bottom of <head> tag and just before </body> would be great

**Comment by Rikk** [comment]
You can do it with JS (UserScript).
Anyway, I think development stopped (or, I think this is the meaning of 1 year without any new commit).

**Comment by FluffyDiscord** [comment]
> 
> 
> You can do it with JS (UserScript).
> Anyway, I think development stopped (or, I think this is the meaning of 1 year without any new commit).

Yes I am aware of that and I am using it that way currently, but still, it would be sweet to have.

I hope dev is still around. Would you mind recommending alternatives which are as fast as this amazing browser and are still in development adding new features ?

**Comment by Rikk** [comment]
I'm still using Yuzu and haven't still found a qualified replacement. However, I'm also using a minimalist browser (900kb apk) which has a Desktop mode that works better (Via Browser, mark.via.gp)

**Comment by Avrution** [comment]
Yuzu development is 99.9% dead, so it is best to move on.  Berry browser is a fork and is turning out tons of releases.  

https://play.google.com/store/apps/details?id=jp.ejimax.yuzubrowser

**Comment by Rikk** [comment]
> Yuzu development is 99.9% dead, so it is best to move on. Berry browser is a fork and is turning out tons of releases.
> https://play.google.com/store/apps/details?id=jp.ejimax.yuzubrowser

Where is its updated code? I found only https://github.com/Ejimax/YuzuBrowser which is just the current Yuzu code.
Other than borrowing the name of another browser app (https://apps.apple.com/us/app/berry-browser/id1411079643), not having public source code is not a good practice for a fork.

**Comment by ksuuk** [comment]
> Yuzu development is 99.9% dead, so it is best to move on. Berry browser is a fork and is turning out tons of releases.
> 
> https://play.google.com/store/apps/details?id=jp.ejimax.yuzubrowser

Like I said, once, Berry is based some very old Yuzu code, containing a lot old Yuzu bugs and the some cosmetic changes in settings, but as code isn't public and developer won't reply, then I don't trust it at all.

**Comment by ksuuk** [comment]
Read also comments about Berry in this thread - https://github.com/hazuki0x0/YuzuBrowser/issues/177

**Comment by krystian3w** [comment]
Linked app for iOS/iPad OS (https://github.com/hazuki0x0/YuzuBrowser/issues/185#issuecomment-661663654) may no base at all on Yuzu/Habit = it is probably only the same name coincidence.

**Comment by nymgmail** [comment]
Help for this please.
Cannot find how to get microphone for search


Spent two hours looking  
Yours is only browser no microphone.

Jack








----- Original message -----

**Comment by FluffyDiscord** [comment]
> > Yuzu development is 99.9% dead, so it is best to move on. Berry browser is a fork and is turning out tons of releases.
> > https://play.google.com/store/apps/details?id=jp.ejimax.yuzubrowser
> 
> Like I said before, Berry is based some very old Yuzu code, containing a lot old Yuzu bugs and is having some cosmetic changes in the settings, but as code isn't public and developer won't reply, then I don't trust it at all.

Also berry browser is terribly slow compared to Yuzu. I have yet to find any good replacement for Yuzu and I am scared that there will be time, when it fails to run one day after random system update

**Comment by ksuuk** [comment]
> > > Yuzu development is 99.9% dead, so it is best to move on. Berry browser is a fork and is turning out tons of releases.
> > > https://play.google.com/store/apps/details?id=jp.ejimax.yuzubrowser
> > 
> > 
> > Like I said before, Berry is based some very old Yuzu code, containing a lot old Yuzu bugs and is having some cosmetic changes in the settings, but as code isn't public and developer won't reply, then I don't trust it at all.
> 
> Also berry browser is terribly slow compared to Yuzu. I have yet to find any good replacement for Yuzu and I am scared that there will be time, when it fails to run one day after random system update


In addition, Berry is closed source and the developer is arrogant, so I'm also constantly looking for Yuzu based open source browser.

**Comment by nymgmail** [comment]
Found a couple at github.......
How to save as file in userscript area for via please?

Merci!!!!!!

**Comment by nymgmail** [comment]
Sorry.....cannot change downloads location dir, please help. Keep pressing to re-pick where I want it.

----- Original message -----
